misc/acrn-config/schema/VMtypes.xsd

```diff
@@ -63,8 +69,16 @@
   <xs:sequence>
 <xs:element name="start_hpa" type="HexFormat" />
 <xs:element name="size" type="MemorySizeType" />
-<xs:element name="start_hpa2" type="HexFormat" minOccurs="0" />
-<xs:element name="size_hpa2" type="MemorySizeType" minOccurs="0" />
+<xs:element name="start_hpa2" type="HexFormat" minOccurs="0">
+  <xs:annotation acrn:configurable="0">
+    <xs:documentation>Start of second HPA for non-contiguous allocations in host for the VM.</xs:documentation>
+  </xs:annotation>
+</xs:element>
+<xs:element name="size_hpa2" type="MemorySizeType" minOccurs="0">
+  <xs:annotation acrn:configurable="0">
```

Comment:
It is a boolean value, we follow the original xml format to make the element not configurable by acrn:configurable="0".
@szhen11 can you confirm if this would work if it changes from acrn:configurable="0" to acrn:configurable="n" or acrn:configurable="false"?

Reply:
it is OK for me. I will update the UI part for this change.

Reply:
Thanks, @szhen11.
@dbkinder have made all acrn:configurable="y"